app/src/main/res/values/strings.xml

```diff
@@ -225,16 +225,16 @@
     <string name="the_number_of_days_allowed">The number of days allowed for vacations may not be left blank!</string>
     <string name="the_field_required_can_not_be_blank">The field required can not be blank!</string>
     <string name="time_must_be_a_working_date">Time must be a working day!</string>
-    <string name="annual_leave_is_not_greater_than_14_5_day">Annual leave is not greater than 14.5 days !</string>
-    <string name="leave_for_marriage_is_not_greater_than_3_day">Leave for marriage is not greater than 3 days !</string>
-    <string name="leave_for_child_marriage_is_not_greater_than_1_day">Leave for child\'s marriage is not greater than 1 day !</string>
-    <string name="funeral_leave_is_not_greater_than_3_day">Funeral leave is not greater than 3 days !</string>
-    <string name="leave_for_care_of_sick_is_not_greater_than_20_day">Leave for care of sick child is not greater than 20 days !</string>
-    <string name="pregnacy_examination_leave_is_not_greater_than_14_5_day">Pregnancy examination leave is not greater than 5 days !</string>
-    <string name="sick_leave_is_not_greater_than_60_day">Sick leave is not greater than 60 days !</string>
-    <string name="miscarriage_leave_is_not_greater_than_50_day">Miscarriage leave is not greater than 50 days !</string>
-    <string name="maternity_leave_is_not_greater_than_180_day">Maternity leave is not greater than 180 days !</string>
-    <string name="wife_labor_leave_is_not_greater_than_14_day">Wife\'s labor leave is not greater than 14 days !</string>
+    <string name="annual_leave_error">Annual leave is not greater than 14.5 days !</string>
+    <string name="leave_for_marriage_error">Leave for marriage is not greater than %s days !</string>
+    <string name="leave_for_child_marriage_error">Leave for child\'s marriage is not greater than %s day !</string>
+    <string name="funeral_leave_error">Funeral leave is not greater than %s days !</string>
+    <string name="leave_for_care_of_sick_error">Leave for care of sick child is not greater than %s days !</string>
+    <string name="pregnacy_examination_leave_error">Pregnancy examination leave is not greater than %s days !</string>
+    <string name="sick_leave_error">Sick leave is not greater than %s days !</string>
+    <string name="miscarriage_leave_error">Miscarriage leave is not greater than %s days !</string>
+    <string name="maternity_leave_error">Maternity leave is not greater than %s days !</string>
+    <string name="wife_labor_leave_error">Wife\'s labor leave is not greater than %s days !</string>
```

Comment:
FATAL: [MissingTranslation] "`wife_labor_leave_error`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)